packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.scss

```diff
@@ -131,12 +134,11 @@ $SearchBox-sidePadding: 8px; // padding in input on left and right sides without
   .ms-CommandBarItem-link {
     @include CommandBarItem-text;
     cursor: pointer;
-
     &:hover,
     &.is-expanded {
       @media screen and (-ms-high-contrast: active) {
-        @include highContrastBorder($ms-color-white);
-      }
+         @include highContrastBorder($ms-color-white);
```

Comment:
Can you double-check the indentation here (2 spaces)? I'm not sure why GitHub is showing changes on these lines.